TestCases/optimization_euler/equivalentarea_naca64206/NACA64206.cfg

```diff
@@ -0,0 +1,434 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: NACA64-206 in inviscid, supersonic flow                  %
+% Author: Yuki Utsumi	      	      	              	       	       %
+% Institution: Individual				               %
+% Date: 2021.07.19                                                             %
+% File Version 7.1.1 "Blackbird"                                                %
+%                                                                              %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+% ------------- DIRECT, ADJOINT, AND LINEARIZED PROBLEM DEFINITION ------------%
+%
+% Physical governing equations (EULER, NAVIER_STOKES)
+SOLVER= EULER
+%
+% Mathematical problem (DIRECT, CONTINUOUS_ADJOINT)
+MATH_PROBLEM= DIRECT
+%
+% Restart solution (NO, YES)
+RESTART_SOL= NO 
+%
+% Write binary restart files (YES, NO)
+%WRT_BINARY_RESTART= NO
+%Commented out for v7
+%
+% Read binary restart files (YES, NO)
+READ_BINARY_RESTART= NO
+
+% -------------------- COMPRESSIBLE FREE-STREAM DEFINITION --------------------%
+%
+% Mach number (non-dimensional, based on the free-stream values)
+MACH_NUMBER= 1.6
+%
+% Angle of attack (degrees)
+AOA= 0.0
+%
+% Side-slip angle (degrees)
+SIDESLIP_ANGLE= 0.0
+%
+% Free-stream pressure (101325.0 N/m^2 by default, only for Euler equations)
+FREESTREAM_PRESSURE= 11662.0
+%
+% Free-stream temperature (288.15 K by default)
+FREESTREAM_TEMPERATURE= 216.6499
+
+% ---------------------- REFERENCE VALUE DEFINITION ---------------------------%
+%
+% Reference origin for moment computation
+REF_ORIGIN_MOMENT_X = 0.25
+REF_ORIGIN_MOMENT_Y = 0.00
+REF_ORIGIN_MOMENT_Z = 0.00
+%
+% Reference length for pitching, rolling, and yaMAIN_BOX non-dimensional moment
+REF_LENGTH= 1
+%
+% Reference area for force coefficients (0 implies automatic calculation)
+REF_AREA= 0
+%
+% Flow non-dimensionalization (DIMENSIONAL, FREESTREAM_PRESS_EQ_ONE,
+%                              FREESTREAM_VEL_EQ_MACH, FREESTREAM_VEL_EQ_ONE)
+REF_DIMENSIONALIZATION= FREESTREAM_PRESS_EQ_ONE
```

Comment:
```suggestion
REF_ORIGIN_MOMENT_X = 0.25
REF_ORIGIN_MOMENT_Y = 0.00
REF_ORIGIN_MOMENT_Z = 0.00
REF_LENGTH= 1
REF_AREA= 0
REF_DIMENSIONALIZATION= FREESTREAM_PRESS_EQ_ONE
```
For example

Reply:
See below.
https://github.com/su2code/SU2/pull/1329#discussion_r673786199